src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -86,8 +90,20 @@ case class EdmWebResource(
   */
 
 case class OreAggregation(
+                           id: ExactlyOne[String],
+                           /*
+                            * FIXME: it's not clear what `uri' corresponds to
+                            * in Section 4.1.C of
+                            * http://dp.la/info/wp-content/uploads/2015/03/MAPv4.pdf
+                            */
                            uri: ExactlyOne[URI], //uri of the record on our site
                            dataProvider: ExactlyOne[EdmAgent],
+                           /*
```

Comment:
I think the OR should be a literal since we're no longer using LDP. Also it's more useful in general for QA and analytics. Doesn't have to be so in output RDF, but otherwise I'd argue it should, particularly since there are no URIs available for many of these records.